tiny-java-containers/jwebserver/Dockerfile.distroless-java-base.dynamic

```diff
@@ -1,5 +1,11 @@
+FROM container-registry.oracle.com/graalvm/native-image:23 AS nativebuild
+WORKDIR /build
+# Build a dynamically linked native executable optimizing for size with `-Os`
+RUN native-image -Os --enable-sbom=cyclonedx -m jdk.httpserver -o jwebserver.dynamic
```

Comment:
We don't explain the `--enable-sbom=cyclonedx ` option in the text. Do you need to add it also to the full static Dockerfile build?

Reply:
Oh, I'll remove that for now.